src/palette.ml

```diff
@@ -43,6 +43,27 @@ let load_tic80_palette (raw : string) : t =
   else
     raise (Invalid_argument "Palette size must not be zero or negative")
 
+(* New function for Task #27 *)
+let load_lospec_palette (s : string) : t =
+  let lines = String.split_on_char '\n' s in
+  let parse_hex line =
+    let line = String.trim line in
+    let hex =
+      if String.length line = 6 then line
```

Comment:
Whilst this works, it isn't very idiomatic OCaml - can you rewrite this using `match`?